Presentation.Web/app/components/org/structure/org-structure.controller.ts

```diff
@@ -632,17 +634,59 @@
                 loadRights($scope.chosenOrgUnit);
             });
 
-            $scope.dragEnabled = false;
+            $scope.isReordering = false;
+            $scope.loadingAccessRights = false;
 
             $scope.toggleDrag = function () {
-                $scope.dragEnabled = !$scope.dragEnabled;
+                $scope.isReordering = !$scope.isReordering;
+
+                if ($scope.isReordering) {
+                    $scope.loadingAccessRights = true;
+                    organizationUnitService.getUnitAccessRightsForOrganization(rootNodeOfOrganization.organization.uuid)
+                        .then(response => {
+                            applyAccessRights(rootNodeOfOrganization, response);
+                            $scope.loadingAccessRights = false;
+                        }, error => {
+                            notify.addErrorMessage("Failed to load unit access rights");
```

Comment:
change text to danish

Reply:
Kunne ikke indlæse rettighederne for organisationsenheden.